src/init.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const Promise = require('bluebird')
+const fs = Promise.promisifyAll(require('fs'))
+const input = require('input')
+
+// Initial config template
+const configTemplate = (image) => `from: ${image}
+services: []
+env: []
+volumes: []
+expose: []
+hosts: []
+before:
+  - echo "before..."
+after:
+  - echo "after..."
+tasks:
+  env: env | sort
+  shell: /bin/sh
+`
+
+/**
+ * Initializes a project by creating a devlab.yml config
+ */
+const init = () => input.text('Enter base docker image to use: ')
+  .then((image) => fs.writeFileAsync(`${process.cwd()}/devlab.yml`, configTemplate(image))
+    .then(() => 'Config file created')
+    .catch(() => new Error('Unable to write config file'))
+  ).catch(() => new Error('Could not read input'))
```

Comment:
Code has been updated and patch released: [v.3.11.1](https://github.com/TechnologyAdvice/DevLab/releases/tag/v3.11.1)